karma.conf.js

```diff
@@ -46,6 +46,7 @@ module.exports = function(config) {
         exclude: /test\//,
       },
       compilerOptions: {
+        module: 'commonjs',
```

Comment:
In `tsconfigs/browser-module` the value for `module` is `ESNEXT`. I don't know what Karma's module system does or expects, but it seems that it does not expect ES modules. So CommonJS it is.